### ISSUE TITLE
Retrieving Historical 'Financials' Data (e.g. quarterly earnings) with Yahoo Finance Premium Subscription?

### ISSUE THREAD

**Issue by lukerbs** [ranaroussi/yfinance#1966]
Hello, is there any functionality in place to leverage Yahoo Finance premium subscription features via yfinance? I have a gold membership and would like to access the additional data via yfinance. Thanks! Awesome project by the way.

**Comment by jab2727** [comment]
Quick follow up, did you ever find a solution to this?

**Comment by dokson** [comment]
yfinance already supports passing a logged-in Yahoo session via `yf.Auth().set_login_cookies(cookie_t, cookie_y)` (see `yfinance/data.py`). However, no scraper currently does anything different for authenticated users — the same endpoints/params are used for everyone. If you have a Gold subscription and can test what changes when authenticated (extended history window, extra fields, different endpoints), a write-up here would unblock a PR.

**Comment by ValueRaider** [comment]
> yfinance already supports passing a logged-in Yahoo session via `yf.Auth().set_login_cookies(cookie_t, cookie_y)`

Not true! It has not been released yet!

**Comment by dokson** [comment]
You're right, my mistake: `Auth` class is only on `dev`, not in any released version yet, so pip users can't access it today.

**Comment by ValueRaider** [comment]
This is released now. See docs
